src/DeviceFirmata.cpp

```diff
@@ -0,0 +1,264 @@
+/*
```

Comment:
This comment became issue finson/FirmataWithDeviceFeature/issues/9 and was closed with the introduction of Optional Features.